man/readSupplycurveBio.Rd

```diff
@@ -6,7 +6,9 @@
 \usage{
 readSupplycurveBio(
   outputdirs,
-  userfun = function(param, x) {     return(param[[1]] + param[[2]] * x) },
+  userfun = function(param, x) {
```

Comment:
Is that intended?

Reply:
it is just two line breaks added, right?

Reply:
The Rd changes (as well as git related ones) are created automatically by the lucode2 buildLibrary code.
I asked RSE to remove the inclusion of these changes in content related PRs. 
However, after discussing this in the mattermost group, there was no common agreement at the time.